config/initializers/scholarsphere_admin.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ScholarsphereAdmin
+  def self.matches?(request)
+    current_user = request.env['warden'].user
+    return false if current_user.blank?
+
+    current_user.admin?
+  end
```

Comment:
@whereismyjetpack this isn't really tested. We're testing `admin?`, which is good, but we ought to have a test the exercises this block. A request spec might work here. Hit the endpoint as an admin and verify it's successful, then hit it as a non-admin and it should be 404, 403, or?

Reply:
Excellent! I'll certainly add another spec

Reply:
I added some request specs that exercise 3 cases on this endpoint. logged out, logged in, and logged in as admin.